src/appshell/qml/ProjectPage/ProjectPage.qml

```diff
@@ -163,23 +172,36 @@ DockPage {
 
             navigationSection: root.navigationPanelSec(tracksPanel.location)
 
-            width: root.verticalPanelDefaultWidth
-            minimumWidth: root.verticalPanelDefaultWidth
-            maximumWidth: root.verticalPanelDefaultWidth
+            width: root.verticalPanelDefaultWidth + (showEffectsSection ? effectsSectionWidth : 0)
+            minimumWidth: width
+            maximumWidth: width
 
             groupName: root.verticalPanelsGroup
 
             dropDestinations: root.verticalPanelDropDestinations
 
             titleBar: TracksTitleBar {
+                id: titleBarItem
+                effectsSectionWidth: tracksPanel.effectsSectionWidth
+                showEffectsSection: tracksPanel.showEffectsSection
+
                 onAddRequested: function(type) {
                     tracksPanel.add(type)
                 }
+                onEffectsSectionCloseButtonClicked: {
```

Comment:
please add new line before this line

Reply:
Line 85 of this file is `onInited: {`, and git tells me you're the author.

Reply:
we all make mistakes

Reply:
@Eism  By the way, what our codestyle says about arrow function syntax?

Reply:
Using functions is necessary to resolve warnings in qml so that we can use the arrow function because it is also a function :)

Reply:
I'll pay attention to this warning, thanks for the info.